command/cat.go

```diff
@@ -71,17 +64,17 @@ var CatCommand = &cli.Command{
 			return err
 		}
 
-		return Cat(c.Context, src, c.Int64("part-size")*megabytes)
+		return Cat(c.Context, src)
 	},
 }
 
-func Cat(ctx context.Context, src *url.URL, partSize int64) error {
+func Cat(ctx context.Context, src *url.URL) error {
 	client, err := storage.NewClient(src)
 	if err != nil {
 		return err
 	}
 
-	_, err = client.Get(ctx, src, sequentialWriterAt{w: os.Stdout}, 1, partSize)
+	_, err = client.Get(ctx, src, sequentialWriterAt{w: os.Stdout}, 1, -1)
```

Comment:
a comment would be nice. what is 1, what is -1, and why.

Reply:
added